seatunnel-connectors-v2/connector-kudu/src/main/java/org/apache/seatunnel/connectors/seatunnel/kudu/kuduclient/KuduOutputFormat.java

```diff
@@ -109,10 +112,10 @@ private void transform(PartialRow row, SeaTunnelRow element, Schema schema) {
                         row.addDecimal(columnIndex, (BigDecimal) element.getField(columnIndex));
                         break;
                     default:
-                        throw new IllegalArgumentException("Unsupported column type: " + col.getType());
+                        throw new KuduConnectorException(KuduConnectorErrorCode.TRANSFORM_TO_KUDU_DATA_TYPE_FAILED, "Unsupported column type: " + col.getType());
```

Comment:
CommonErrorCode.UNSUPPORTED_DATA_TYPE is better, remove this unclear error code.